### PR TITLE
Fix/sizing #10

### PR DESCRIPTION
**Pull request by Muhammad-Shahid-Jamal**
# Screen Shot
Screenshot 2023-10-22 at 12 02 43 PM

### Changes
- [x] padding and margin are the same as other sections
- [x] follow coding standards to do it.
- [x] package-lock.json file added to gitignore
- [x] pnpm-lock.yaml file added to gitignore   
- [x] unused import removed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,5 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 .vercel
+package-lock.json
```
Comment (tamalCodes) [inline .gitignore]: Please revert this change.

File: src/sections/ThemesSection/ThemesSection.jsx [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
-          >
-            To be decided soon...
-          </h3>
+    <motion.div
+      initial={{ opacity: 0 }}
+      whileInView={{ opacity: 1, transition: { duration: 1 } }}
+      viewport={{ once: false }}
+      className="hack4bengal__section5"
+    >
+      <div className="h4b_about_main">
```
Comment (tamalCodes) [inline src/sections/ThemesSection/ThemesSection.jsx]: As of the issue mentioned i don't think we might be needing framer motion's tool to get it done, i guess we can revert this to have a normal div in place.

**Review by tamalCodes — CHANGES_REQUESTED** [review]

**Review by Muhammad-Shahid-Jamal — COMMENTED** [review]
I have reverted gitignore and replaced simple div

**Review by tamalCodes — APPROVED** [review]
Great work, thanks for making the changes !

**Pull request merged**